superset-frontend/src/explore/components/RowCountLabel.stories.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import RowCountLabel from './RowCountLabel';
+
+export default {
+  title: 'RowCountLabel',
+  component: RowCountLabel,
+};
+
+const options = {
+  loading: {
+    loading: true,
```

Comment:
I think we need to either add `rowcount: 0` here or set `rowcount = 0` as a default value in the component. Now we get a red "danger" colour in storybook, which shouldn't happen.